### ISSUE TITLE
here is some wrong run in my mac 

### ISSUE THREAD

**Issue by 1980744819** [sadeqa/Super-Mario-Bros-RL#2]
I want to know what platform  can this code run? eg:ubuntu/macos ,python 3.7/2.7 ,and which version of each lib/framework????

**Comment by otmhi** [comment]
Hi, we mainly experimented on ubuntu and windows with python 3.6. maybe the latest versions of the libraries may output some deprecation warnings but everything runned smoothly.

Can you give us the real issue encountered? maybe the throwed errors?

**Comment by 1980744819** [comment]
> Hi, we mainly experimented on ubuntu and windows with python 3.6. maybe the latest versions of the libraries may output some deprecation warnings but everything runned smoothly.
> 
> Can you give us the real issue encountered? maybe the throwed errors?

thank you, I will try to run it on Ubuntu,and I really want some example ordirections of use,maybe I just run it with  wrong commands. best wishes.

**Comment by otmhi** [comment]
> > Hi, we mainly experimented on ubuntu and windows with python 3.6. maybe the latest versions of the libraries may output some deprecation warnings but everything runned smoothly.
> > Can you give us the real issue encountered? maybe the throwed errors?
> 
> thank you, I will try to run it on Ubuntu,and I really want some example ordirections of use,maybe I just run it with wrong commands. best wishes.

Yes of course, the readme in each of A3C and PPO folders explain how to launch the script.

For instance, using A3C ( which we recommend ), you can launch the script 'python3 train-mario.py --reward_type <dense or sparse>' for a normal agent and 'python3 train-mario-curiosity.py --reward_type <dense or sparse>' for a curious agent.

Hope this helps.
